packages/graph/hash_graph/lib/graph/src/types/schema/combinator.rs

```diff
@@ -16,22 +9,6 @@ pub enum Optional<T> {
     Some(T),
 }
```

Comment:
`Optional` is not used anymore, let's delete it.

Reply:
Good spot, surprised Clippy didn't warn me. I guess because of the tests